integration-test/src/main/java/com/sequenceiq/it/cloudbreak/action/sdx/SdxBackupAction.java

```diff
@@ -33,7 +33,7 @@ public SdxBackupAction(String backupLocation, String backupName) {
     public SdxTestDto action(TestContext testContext, SdxTestDto testDto, SdxClient client) throws Exception {
         String sdxName = testDto.getName();
 
-        sleep(2, sdxName);
+        sleep(3, sdxName);
```

Comment:
Also code might be more readable with passing in a TimeUnit or updatating the refactored method name to `sleepForMinutes(...)`